### DISCUSSION TITLE
Add tags with the keyboard (in addition to scrolling)

### DISCUSSION THREAD

**Discussion by ronaldheft** [LeoKlaus/Paperparrot#87] [Feature Requests]
Currently the tag selection UI requires scrolling through a list to select tags. If you have a lot of tags, this UI element can be cumbersome as tags toward the end require a lot of scrolling.

It would be nice if you could select a tag by typing part of its name using the keyboard. A good example of this is the web-ui for paperless-ngx.

**Comment by rjbutler** [comment]
It would also be nice if you could more easily delete tags without have to find it in the drop down and click on it.  Similar to the web UI.

**Comment by LeoKlaus** [comment]
While I agree that this is very helpful when working with a large list of tags, something like this is not possible with native elements in SwiftUI. Building a component like this is certainly possible, but it'll be hard to make it feel native in iOS/macOS.
I'll see what I can build, but I can't make any promises, especially for the iOS version.

**Comment by LeoKlaus** [comment]
I've just added new menus for all attributes in the latest TestFlight release, so I'll close this.
